packages/malloy/src/lang/ast/source-elements/named-source.ts

```diff
@@ -196,6 +197,14 @@ export class NamedSource extends Source {
           parameterSpace ?? new ParameterSpace(parametersOut ?? []);
         const pVal = argument.value.getExpression(paramSpace);
         let value = pVal.value;
+        if (
+          pVal.type === 'filter expression' &&
+          parameter.type === 'filter expression' &&
+          parameter.filterType
+        ) {
+          // mtoy todo ask chris how to get a better error reporting object
```

Comment:
For sure want to talk to chris about this before I merge

Reply:
I think the thing you want to log to here is `argument`

Reply:
Or perhaps `argument.value`

Reply:
I was thinking I wanted the error location to be the location of the source of the filter expression.

I guess putting the actual filter expression in the error and making the location still be the entire source is ok for MVP

?